packages/lib/server-only/document/create-document.ts

```diff
@@ -1,19 +1,27 @@
 'use server';
 
 import { prisma } from '@documenso/prisma';
+import type { DocumentStatus } from '@documenso/prisma/client';
 
 export type CreateDocumentOptions = {
   title: string;
   userId: number;
   documentDataId: string;
+  status?: DocumentStatus;
 };
 
-export const createDocument = async ({ userId, title, documentDataId }: CreateDocumentOptions) => {
+export const createDocument = async ({
```

Comment:
added `status` arg because this function can be reuseable in doxument test fixture to create documents based on status